crates/wasi-http/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+    name        = "wasi-http"
+    version     = "0.0.1"
+    authors.workspace = true
+    edition.workspace = true
+    repository  = "https://github.com/WebAssembly/wasi-http"
+    license     = "Apache-2.0 WITH LLVM-exception"
+    description = "Experimental HTTP library for WebAssembly in Wasmtime"
+    readme      = "readme.md"
+
+[dependencies]
+    anyhow = { workspace = true }
+    bytes = "1.1.0"
+    hyper = { version = "1.0.0-rc.3", features = ["full"] }
+    tokio = { version = "1", features = ["full"] }
+    tokio-native-tls = { version = "0.3.1" }
+    futures = { version = "0.3.27" }
+    http-body = "0.1.0-rc.2"
```

i believe this was a typo
```suggestion
    http-body = "1.0.0-rc.2"
```